packages/client/src/containers/LandingPage/AddTeamMemberModal.jsx

```diff
@@ -14,26 +16,33 @@ export const AddTeamMemberModal = ({
     return null;
   }
   return (
-    <div>
-      <h2>Add Team Member</h2>
-      <input
-        type="text"
-        value={newMemberFirstName}
-        onChange={(e) => setNewMemberFirstName(e.target.value)}
-        placeholder="First Name"
-      />
-      <input
-        type="text"
-        value={newMemberLastName}
-        onChange={(e) => setNewMemberLastName(e.target.value)}
-        placeholder="Last Name"
-      />
-      <button type="button" onClick={handleAddMember}>
-        Add Member
-      </button>
-      <button type="button" onClick={() => setShowAddModal(false)}>
-        Cancel
-      </button>
+    <div className="container">
```

Comment:
The className is too generic and could cause problems later, which might be difficult to debug, as you will forget this naming logic once working on something else. 
It is a better practice to always have an indicative noun, f.x. "add-member-container". 🙏🏽